docs/content/querying/filters.md

```diff
@@ -58,18 +58,28 @@ The filter specified at field can be any other filter defined on this page.
 
 ### JavaScript filter
 
-The JavaScript filter matches a dimension against the specified JavaScript function predicate. The filter matches values for which the function returns true.
+The JavaScript filter matches dimensions against the specified JavaScript function predicate. The filter matches values for which the function returns true.
 
-The function takes a single argument, the dimension value, and returns either true or false.
+The function takes same number of arguments as the dimension values, and returns either true or false.
 
 ```json
 {
   "type" : "javascript",
-  "dimension" : <dimension_string>,
-  "function" : "function(value) { <...> }"
+  "dimension" : <a dimension_string>,
+  "dimensions" : <array of dimension_strings>,
+  "function" : "function(value1, value2, ...) { <...> }"
+  "rowBased" : "false"
```

Comment:
i feel we need a better name than 'rowBased' for what this flag is doing, but I also don't have any ideas